product-scenarios/1-manage-users-and-roles/1.1-admin-user-manage-roles/1.1.1-manage-roles-with-scim2/src/test/java/org.wso2.identity.scenarios.test.scim2/RoleNamingViolationSCIM2TestCase.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.identity.scenarios.test.scim2;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.json.simple.JSONObject;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import org.wso2.identity.scenarios.commons.SCIM2CommonClient;
+import org.wso2.identity.scenarios.commons.ScenarioTestBase;
+
+import static org.testng.Assert.assertEquals;
+import static org.wso2.identity.scenarios.commons.util.Constants.IS_HTTPS_URL;
+
+public class RoleNamingViolationSCIM2TestCase extends ScenarioTestBase {
```

Comment:
Shall we add a comment for the class and all the other test classes in this PR

Reply:
addressed with https://github.com/wso2/product-is/pull/6381